CONTRIBUTING.md

```diff
@@ -1,65 +1,99 @@
+  - **Idle**: This label will be **automatically** applied to both issues/PRs to indicate that no activity has been observed on the issue/PR for some determined time frame. This an trigger an action to close the issue/PR.
+  - **Unresolved**: This label will be **automatically** applied to PRs to indicate that no consensus has been reached on the PR. This triggers an action to resolve the PR by the TSC.
+  - **Forced**: This label will be **automatically** applied to PRs to indicate that the PR has been merged without following the default process (due to reasons identified below)
 
 <a name="issues"></a>
-### Issues & Labels
-- Issue Types:
-  - **Roadmap Item**: This issue type indicates a request for feature to be added for an upcoming release. 
-    - This issue type should only be created by Technical Steering Leads.
-  - **Fix needed**: This issue type indicates a request to fix a bug or to introduce an enhancement.
-    - This issue type can be created by any individual.
-  - **Backlog**: This issue type indicate a feature request (to be considered during technical steering meeting) for future releases.
-    - This issue type can be created by any individual.
-  - **Enhancement**: This issue type indicate an enhancement request
-    - This issue type can be created by any individual.
-- All issues must be clearly titled, described, and tagged with the right labels when created.
-- Issue Labels (one or more to be selected form pre-defined set):
-  - **Roadmap Item**: To indicate this issue is for a roadmap item.
-  - **Fix needed**: To indicate this issue is for a fix request.
-  - **Backlog**: To indicate this issue is for a backlog feature request.
-  - **Workstream Name**: To indicate which project this issue targeting.
-  - **Chapter Number**: To indicate which Chapter this issue targeting.
-  - **General**: To indicate that this issue is not corresponding to any particular chapter/file.
-  - **Major/Minor**: To indicate if the issue request a major or minor change. 
-  - **Enhancement**: Suggested Enhancement.
+### Issues
 
+- All issues must be clearly titled, described, and tagged with the right labels when created.
+  - Title needs to be in the following format [WSx Chx].
+    - WSx: The name of the work stream it is targeting.
+    - Chx: The name of the chapter it is targeting (if applicable).
+- Issues should be correctly set with the correct Project and Milestone.
+  - Milestone = the upcoming release related to the issue/PR.
+  - Project = Work stream.
+- Workstream Leads has the sole responsibility to assign issues to individuals to address them.
+- **self assignment** are NOT recommended. Contributors may ask WSL (vai comment or others) to get the issue assigned to them if they would like to work on it.
+  - This is to make sure items are worked on based on priorities and right reasoning.
```

Comment:
tpyo: extra "on" in "...  are worked on based on .." ---> "... are worked based on  ..."

Reply:
fixed